packages/client/src/components/FooterSection/FooterSection.component.js

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
```

Comment:
I'm a little confused as to why call it `FooterSection` rather than just `Footer`?